litellm/utils.py

```diff
@@ -4739,6 +4739,8 @@ def _map_and_modify_arg(supported_params: dict, provider: str, model: str):
             optional_params["stop_sequences"] = stop
         if tools is not None:
             optional_params["tools"] = tools
+        if seed is not None:
+            optional_params["seed"] = seed
```

Comment:
this needs to be added to `get_supported_openai_params` in `utils.py` too for `cohere_chat`

Reply:
nvm I see you added it too